assets/scss/_videos_project.scss

```
@@ -90,20 +90,4 @@
-    iframe {
-      position: absolute;
-      top: 0;
-      left: 0;
-      width: 100%;
-      height: 100%;
-      border:0;
-    }
-
-}
\ No newline at end of file
+@import "yt-embed-container";
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)